lib/myapp_web/live/product_live/form_component.ex

```diff
@@ -22,10 +31,20 @@ defmodule MyappWeb.ProductLive.FormComponent do
       |> Map.put(:action, :validate)
 
     Task.async(fn ->
-      HTTPoison.get('http://slowwly.robertomurray.co.uk/delay/2000/url/http://www.google.co.uk')
+      result =
+        HTTPoison.get('http://slowwly.robertomurray.co.uk/delay/2000/url/http://www.google.co.uk')
+
+      Process.sleep(500)
+
+      {:result, socket.assigns.myself, result}
```

Comment:
This bit is important. We are returning `socket.assigns.myself` from the async function, so that the parent live view knows who to route the changes to.